h/static/scripts/annotator/selections.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+var Observable = require('zen-observable');
+
+/**
+ * Returns an observable of events emitted by `src`.
+ *
+ * @param {EventTarget} src - The event source.
+ * @param {Array<string>} eventNames - List of events to subscribe to
+ */
+function listen(src, eventNames) {
```

Comment:
Note: The `listen()`, `buffer()` and `merge()` functions in this file are generic utilities for creating and working with observables. It would probably make sense to extract them into a separate file.